apps/web/src/pages/TalentNominations/NominationGroup/CareerExperience.tsx

```diff
@@ -1,40 +1,185 @@
 import { useQuery } from "urql";
+import useIntl from "react-intl/src/components/useIntl";
+import FlagIcon from "@heroicons/react/24/outline/FlagIcon";
 
-import { FragmentType, getFragment, graphql } from "@gc-digital-talent/graphql";
-import { Pending, ThrowNotFound } from "@gc-digital-talent/ui";
+import { graphql } from "@gc-digital-talent/graphql";
+import {
+  CardBasic,
+  Heading,
+  Pending,
+  Separator,
+  ThrowNotFound,
+  Well,
+} from "@gc-digital-talent/ui";
 import { ROLE_NAME } from "@gc-digital-talent/auth";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+import { formatDate, parseDateTimeUtc } from "@gc-digital-talent/date-helpers";
+import { commonMessages } from "@gc-digital-talent/i18n";
 
 import RequireAuth from "~/components/RequireAuth/RequireAuth";
 import useRequiredParams from "~/hooks/useRequiredParams";
+import { isWorkExperience } from "~/utils/experienceUtils";
 
 import { RouteParams } from "./types";
+import CurrentPositionExperiences from "./components/CurrentPositionExperiences";
 
-const TalentNominationGroupCareerExperience_Fragment = graphql(/* GraphQL */ `
-  fragment TalentNominationGroupCareerExperience on TalentNominationGroup {
-    id
+const NomineeExperiences_Query = graphql(/* GraphQL */ `
+  query NomineeExperiences($nomineeId: UUID!) {
+    user(id: $nomineeId) {
+      updatedDate
+      experiences {
+        id
+        __typename
+        # ... on AwardExperience {
+        #   title
+        # }
+        # ... on CommunityExperience {
+        #   title
+        # }
+        # ... on EducationExperience {
+        #   institution
+        # }
+        # ... on PersonalExperience {
+        #   title
+        ... on WorkExperience {
+          ...CurrentPositionWorkExperience
+        }
+      }
+    }
   }
 `);
 
 interface TalentNominationGroupCareerExperienceProps {
-  query: FragmentType<typeof TalentNominationGroupCareerExperience_Fragment>;
+  nomineeId: string;
+  shareProfile?: boolean;
 }
 
 const TalentNominationGroupCareerExperience = ({
-  query,
+  nomineeId,
+  shareProfile,
 }: TalentNominationGroupCareerExperienceProps) => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const talentNominationGroup = getFragment(
-    TalentNominationGroupCareerExperience_Fragment,
-    query,
+  const intl = useIntl();
+
+  const [{ data, fetching, error }] = useQuery({
+    query: NomineeExperiences_Query,
+    variables: { nomineeId },
+    pause: !shareProfile,
+  });
+
+  const lastUpdated = data?.user?.updatedDate
+    ? formatDate({
+        date: parseDateTimeUtc(data.user.updatedDate),
+        formatString: "MMMM d, yyyy",
+        intl,
```

Comment:
Last updated user is easy enough, last time user updated their profile OR experiences not easy

Reply:
@petertgiles

Reply:
To be enhanced in https://github.com/GCTC-NTGC/gc-digital-talent/issues/13351